lampshade_test.go

```diff
@@ -164,7 +164,7 @@ func TestPhysicalConnCloseRemotePrematurely(t *testing.T) {
 	assert.Equal(t, 0, n)
 
 	_, err = conn.Write([]byte("whatever"))
-	assert.Equal(t, ErrBrokenPipe, err)
+	assert.Equal(t, syscall.EPIPE, err)
```

Comment:
`syscall.EPIPE` satisfies `net.Error` as well. The reason why I have to change here is that `err` is always `syscall.EPIPE` even though the `onSessionError` returns `ErrBrokenPipe`. I guess it's because the `readErr` is already `syscall.EPIPE` somehow so `onSessionError` doesn't [rewrite it](https://github.com/getlantern/lampshade/blob/master/session.go#L535). It should be a timing thing.